src/native/configuration/endpoints.rs

```diff
@@ -16,13 +14,11 @@ impl Default for Endpoints {
         if option_env!("DEV_ENDPOINTS").is_some() {
             Self {
                 core: "http://127.0.0.1:3001".to_string(),
-                data: "http://127.0.0.1:3002".to_string(),
-                frontend: "http://127.0.0.1:3000".to_string(),
+                frontend: "http://127.0.0.1:3001".to_string(),
```

Comment:
We can get rid of the frontend as well, this will always be the core service URL as well (Its available at both the api. domain and the beta/alpha domain but that's all combined into one now).